lumicks/pylake/scan.py

```diff
@@ -352,8 +352,16 @@ def lines_per_frame(self):
 
     @property
     def _shape(self):
+        # Note that this is not the shape of the image property, but rather the shape in terms of
+        # [slow axis, fast axis].
```

Comment:
I appreciate the comment, but I still had a hard time to follow why `_shape()` is not the actual shape of the image. If I understand it corretly, `_shape()` returns the original shape of the `raw_image`, i.e. the uncropped image. Would it therefore make sense to rename the property `_shape` to `_raw_image_shape`, `_original_image_shape` or similar? I checked, it would require to change the name in three other places (`confocal._default_image_factory()`, `confocal._default_timestamp_factory()`, and `Kymo._shape`).

Reply:
Nah, it's in a different axis order (slow to fast; reconstruction order). It's purely used in the reconstruction of images from raw photon counts and is an implementational detail in how they are reshaped.

They do respond to cropping actually, since they use properties that change, but it doesn't matter because a cropped/downsampled image is never reconstructed anew.

I hate the name too, so you're not alone. I think `_reconstruction_shape` would maybe be a more sensible name to convey that this is really a property that exists only for reconstruction purposes.

Maybe the docstring should read something like
```python
Shape used when reconstructing the image from raw photon counts (ordered by axis scan speed slow to fast).
``` 

What do you think?